### PR TITLE
changes for the 1.0 release

### PR DESCRIPTION
**Pull request by rsnancollas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,12 +66,24 @@ A digital datasheet shall include the following information:
...
 
-#### 3.3 Date Format
+#### 4.3 Date Format
 A digital datasheet shall follow the international standard notation, YYYY-MM-DD.
 
-### 3.4	 Common
+####  4.4	 Top Level Component Specification 
+
+Source: [component.json](https://github.com/edatasheets/digital-datasheets/blob/main/part-spec/component.json)
+
```
Comment (asadate) [inline README.md:75]: The tool didn't update the links to github.io.

File: examples/ic_io/retimer_PI2EQX22024_output.json [hunk 2]
```diff
@@ -0,0 +1,1786 @@
+{
+    "componentID": {
+        "partType": "retimer",
+        "manufacturer": "DIODES",
+        "componentName": "PI2EQX22024",
+        "orderableMPN": [
+            "PI2EQX22024XEAEX"
+        ],
+        "sourceDatasheetID": {
+            "publishedDate": "December, 2022",
```
Comment (asadate) [inline examples/ic_io/retimer_PI2EQX22024_output.json]: Based on discussion in WG today, we can change the dates in the example to match YYYY-MM-DD to match the specifications. 8-).

File: examples/semiconductor/diode_BAS16_output.json [hunk 3]
```diff
@@ -0,0 +1,272 @@
+{
+    "componentID": {
+        "partType": "diode",
+        "manufacturer": "Nexperia",
+        "componentName": "BAS16",
+        "orderableMPN": [
+            "BAS16"
```
Comment (asadate) [inline examples/semiconductor/diode_BAS16_output.json]: Since we have Nexperia's feedback on BAS16, we can add before releasing 1.0

**Review by asadate — CHANGES_REQUESTED** [review]
Main change: change link to point to github.io instead of digital_datasheets.
Suggested Changes:
1) Change dates in all examples to match our specifications: YYYY-MM-DD
2) Include Herman's feedback on the diode example.

File: README.md (hunk 1 above)
Comment (rsnancollas) [reply README.md:75]: whoops - fixed. thanks

File: examples/semiconductor/diode_BAS16_output.json (hunk 3 above)
Comment (rsnancollas) [reply examples/semiconductor/diode_BAS16_output.json]: done

File: examples/ic_io/retimer_PI2EQX22024_output.json (hunk 2 above)
Comment (rsnancollas) [reply examples/ic_io/retimer_PI2EQX22024_output.json]: done

**Review by asadate — APPROVED** [review]

**Pull request merged**